Content.Server/DeviceLinking/Systems/SignalTimerSystem.cs

```diff
@@ -78,17 +84,22 @@ public override void Update(float frameTime)
 
     private void UpdateTimer()
     {
+        List<EntityUid> uids = new List<EntityUid>();
```

Comment:
why have this list instead of just triggering in the query loop

Reply:
> why have this list instead of just triggering in the query loop
> 
> you mustve have misinterpreted sloths review, the `break;` was stopping updating other timers, just replace it with `continue;` and use old way

I tried using `continue;` but I get an error.  Only using `break;` or list helped, it works perfectly with them. Without them, the error occurs because the values inside MoveNext are replaced in the process. The error occurs if the timer starts another timer. I don't even know what to do if `break;` and list is not suitable.

Reply:
Making a new list every tick is bad but also not just storing the components is also bad as trycomp isn't free.